### DISCUSSION TITLE
Broke my Git

### DISCUSSION THREAD

**Discussion by SquilliamX** [Cyfrin/foundry-full-course-cu#598] [Q&A] [answered]
I was trying to save/upload my smart contract lottery in Github. Had all my contracts in the staging area and when i commited my git with a message, i thought i would see it in my repos, but i can not seem to find it. I changed my named on github about a week ago could that have effected my git commit from the command line? I was able to upload foundry-fund-me-f23 last month but now it seems i broke something and AI is driving me in circles.

 This is what I see in my terminal
 ```
$ git status
On branch master
nothing to commit, working tree clean
 ```
"Git add . " returns nothing
"git remote -v" returns nothing

What commands can I run to find exactly where my repository went?

**Comment by theirrationalone** [comment, marked as answer]
Check your terminal's navigation, it is pointing to right directory?? 

# Or you might not connected to your repo because

`git remote -v` not giving remote url, please try to reconnect your repo using `git remote` command. 

Try this to connect:

```terminal
git remote add origin <remote-url>
```

If you're getting some issues on above command. 

Try this

```terminal
git remote set-url origin <new-remote-url>
```

origin == repo's remote url

then try to execute those commands again... 


git add. 

git status

git commt -m "message"


# A simple and recommended procedure to push a repo on github... 

Remote url: your github's repo https url
branch name: usually it have main Or master. 

```
git init

git add .

git commit -m "commit msg"

git remote add origin <remote-url>

git push origin <branch-name>

```
